collect_app/src/main/java/org/odk/collect/android/formmanagement/download/FormDownloadExceptionMapper.kt

```diff
@@ -17,7 +17,7 @@ class FormDownloadExceptionMapper(private val context: Context) {
             is FormDownloadException.FormParsingError -> {
                 context.getLocalizedString(
                     org.odk.collect.strings.R.string.form_parsing_error
-                ) + " " + context.getLocalizedString(
+                ) + ": ${exception.original.message}. " + context.getLocalizedString(
```

Comment:
For the forum issue, we'd have gotten the message `kotlin.UninitializedPropertyAccessException: lateinit property mainInstanceRoot has not been initialized` right? Would that be enough to figure out what's going on? I was imagining a full strack trace at the bottom but maybe that's overkill.

Reply:
The : and the placeholder for the error message should go in the translated string because they get written differently in different languages.

Reply:
> For the forum issue, we'd have gotten the message kotlin.UninitializedPropertyAccessException: lateinit property mainInstanceRoot has not been initialized right?

Good point! How about including a partial stack trace. Including the message and the first line looks good to me:

![image](https://github.com/user-attachments/assets/a7f11344-b50f-40cf-86a8-505acd27a0a6)

> The : and the placeholder for the error message should go in the translated string because they get written differently in different languages.

I'll work on a proper translated string once we've got the format agreed on 👍.

Reply:
Yes, perfect, thanks!